.env

```diff
@@ -153,8 +153,22 @@ MF_CERTS_SIGN_RSA_BITS=2048
 MF_CERTS_VAULT_HOST=
 MF_CERTS_VAULT_PKI_PATH=pki_int
```

Comment:
default values here are `pki_int_mainflux` and role `mainflux` right? we should put them here than

Reply:
I am going to change these in the next PR which is going to be related to certs service.